pkg/kubeapiserver/admission/configuration/configuration_manager.go

```diff
@@ -26,8 +26,9 @@ import (
 )
 
 const (
-	defaultInterval         = 1 * time.Second
-	defaultFailureThreshold = 5
+	defaultInterval             = 1 * time.Second
+	defaultFailureThreshold     = 5
+	defaultBootstrapGraceperiod = 5 * time.Second
```

Comment:
So the difference here is that this holds from the time the config manager starts running, vs my code which holds from the time the request comes in.  So this is hardcoding in this code that startup takes exactly 5s from the time the admission controller chain starts to the time that the endpoint is available, which doesn't really help.  I expect that time to be variable.